admin-ui/app/locales/en/translation.json

```diff
@@ -169,6 +169,9 @@
     "saml1_uri": "Saml1 URI",
     "saml2_uri": "Saml2 URI",
     "scope_type": "Scope type",
+    "umaAuthorizationPolicies": "Authorization Policies",
+    "umaResourcesScript": "Associated Clients",
```

Comment:
Why `umaResourcesScript` is `Associated Clients`? 

`Associated Clients`  should be obtained scopes obj passed from the list page.

![image](https://user-images.githubusercontent.com/32794267/182845071-66867b5d-0289-4657-85ac-9a7e54951cfd.png)

Reply:
removed